content/docs/command-reference/remote/modify.md

```diff
@@ -250,27 +250,71 @@ For more information on configuring Azure Storage connection strings, visit
 
 Please check out
 [Setup a Google Drive DVC Remote](/doc/user-guide/setup-google-drive-remote) for
-a full guide on configuring Google Drives for use as DVC remote storage,
-including obtaining the necessary credentials, and how to form `gdrive://` URLs.
+a full guide on using Google Drive as DVC remote storage, including how to form
+`gdrive://` URLs, and high-performance/enterprise setups.
 
 - `url` - remote location URL.
 
   ```dvc
   $ dvc remote modify myremote url gdrive://root/path/to/folder
   ```
 
-- `gdrive_client_id` - Google Project's OAuth 2.0 **client ID**.
+- `gdrive_client_id` - Google Project's **client ID** for authentication with
```

Comment:
"Required along with `gdrive_client_secret ` if you use custom app ..." - with a link to the section in the UG

Reply:
Updated